source/module_hamilt_lcao/hamilt_lcaodft/operator_lcao/op_exx_lcao.h

```diff
@@ -53,6 +54,8 @@ class OperatorEXX<OperatorLCAO<TK, TR>> : public OperatorLCAO<TK, TR>
 
       void add_loaded_Hexx(const int ik);
       const K_Vectors& kv;
+
+      LCAO_Matrix* LM = nullptr;
```

Comment:
Is it possible to delete this pointer?

Reply:
Yes, but not in this PR, the variables in LM is only used by EXX now, maybe @PeizeLin or @maki49 can help to refactor this code.